From 0d582293932bc092cdcf08bd4db4becdb632cf29 Mon Sep 17 00:00:00 2001
Message-Id: <0d582293932bc092cdcf08bd4db4becdb632cf29.1368111913.git.minovotn@redhat.com>
In-Reply-To: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
References: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
From: Amit Shah <amit.shah@redhat.com>
Date: Wed, 24 Apr 2013 08:17:49 +0200
Subject: [PATCH 15/65] char: rename qemu_chr_printf() -> qemu_chr_fe_printf()

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <90b51cfe7dc930e8933a6a95ccd8b306e90774aa.1366724981.git.amit.shah@redhat.com>
Patchwork-id: 50793
O-Subject: [RHEL6.5 qemu-kvm PATCH 15/65] char: rename qemu_chr_printf() -> qemu_chr_fe_printf()
Bugzilla: 909059
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: Anthony Liguori <aliguori@us.ibm.com>

Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit e7e71b0ec62d2954635ba8a2462a20a60fa2c147)

Signed-off-by: Amit Shah <amit.shah@redhat.com>

Conflicts:
	qemu-char.h

Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 hw/mips_malta.c | 22 +++++++++++-----------
 qemu-char.c     |  2 +-
 qemu-char.h     |  2 +-
 3 files changed, 13 insertions(+), 13 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/mips_malta.c | 22 +++++++++++-----------
 qemu-char.c     |  2 +-
 qemu-char.h     |  2 +-
 3 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/hw/mips_malta.c b/hw/mips_malta.c
index 6d536e3..5448eac 100644
--- a/hw/mips_malta.c
+++ b/hw/mips_malta.c
@@ -97,8 +97,8 @@ static void malta_fpga_update_display(void *opaque)
     }
     leds_text[8] = '\0';
 
-    qemu_chr_printf(s->display, "\e[H\n\n|\e[32m%-8.8s\e[00m|\r\n", leds_text);
-    qemu_chr_printf(s->display, "\n\n\n\n|\e[31m%-8.8s\e[00m|", s->display_text);
+    qemu_chr_fe_printf(s->display, "\e[H\n\n|\e[32m%-8.8s\e[00m|\r\n", leds_text);
+    qemu_chr_fe_printf(s->display, "\n\n\n\n|\e[31m%-8.8s\e[00m|", s->display_text);
 }
 
 /*
@@ -422,15 +422,15 @@ static void malta_fpga_reset(void *opaque)
 
 static void malta_fpga_led_init(CharDriverState *chr)
 {
-    qemu_chr_printf(chr, "\e[HMalta LEDBAR\r\n");
-    qemu_chr_printf(chr, "+--------+\r\n");
-    qemu_chr_printf(chr, "+        +\r\n");
-    qemu_chr_printf(chr, "+--------+\r\n");
-    qemu_chr_printf(chr, "\n");
-    qemu_chr_printf(chr, "Malta ASCII\r\n");
-    qemu_chr_printf(chr, "+--------+\r\n");
-    qemu_chr_printf(chr, "+        +\r\n");
-    qemu_chr_printf(chr, "+--------+\r\n");
+    qemu_chr_fe_printf(chr, "\e[HMalta LEDBAR\r\n");
+    qemu_chr_fe_printf(chr, "+--------+\r\n");
+    qemu_chr_fe_printf(chr, "+        +\r\n");
+    qemu_chr_fe_printf(chr, "+--------+\r\n");
+    qemu_chr_fe_printf(chr, "\n");
+    qemu_chr_fe_printf(chr, "Malta ASCII\r\n");
+    qemu_chr_fe_printf(chr, "+--------+\r\n");
+    qemu_chr_fe_printf(chr, "+        +\r\n");
+    qemu_chr_fe_printf(chr, "+--------+\r\n");
 }
 
 static MaltaFPGAState *malta_fpga_init(target_phys_addr_t base, qemu_irq uart_irq, CharDriverState *uart_chr)
diff --git a/qemu-char.c b/qemu-char.c
index 8e8befa..011ec3a 100644
--- a/qemu-char.c
+++ b/qemu-char.c
@@ -175,7 +175,7 @@ void qemu_chr_accept_input(CharDriverState *s)
         s->chr_accept_input(s);
 }
 
-void qemu_chr_printf(CharDriverState *s, const char *fmt, ...)
+void qemu_chr_fe_printf(CharDriverState *s, const char *fmt, ...)
 {
     char buf[READ_BUF_LEN];
     va_list ap;
diff --git a/qemu-char.h b/qemu-char.h
index e0eb08a..71d248d 100644
--- a/qemu-char.h
+++ b/qemu-char.h
@@ -84,7 +84,7 @@ void qemu_chr_guest_open(struct CharDriverState *chr);
 void qemu_chr_guest_close(struct CharDriverState *chr);
 void qemu_chr_set_echo(struct CharDriverState *chr, bool echo);
 void qemu_chr_close(CharDriverState *chr);
-void qemu_chr_printf(CharDriverState *s, const char *fmt, ...);
+void qemu_chr_fe_printf(CharDriverState *s, const char *fmt, ...);
 int qemu_chr_fe_write(CharDriverState *s, const uint8_t *buf, int len);
 void qemu_chr_send_event(CharDriverState *s, int event);
 void qemu_chr_add_handlers(CharDriverState *s,
-- 
1.7.11.7

